Algorithms/Algorithms/BInaryHeap/BinaryHeap.cs

```diff
@@ -0,0 +1,114 @@
+using System;
```

Comment:
Please check folder name, it's BInaryHeap instead BinaryHeap

Reply:
Fixed in d04a975